odf_transform/test/test_odf.py

```diff
@@ -0,0 +1,245 @@
+import glob
+import json
+import os
+import sys
+import traceback
+from datetime import datetime
+
+import numpy as np
+from pytz import timezone
+
+sys.path.insert(0, "../../")
+
+from ios_data_transform import is_in
+from ios_data_transform.utils.utils import fix_path
+from odf_transform.odfCls import CtdNcFile
+from odf_transform.utils.utils import get_geo_code, read_geojson
+from odf_transform.utils.oce import get_odf_var_attributes_to_oce, convert_oce_units_to_udunit
+
+
+
+
+CONFIG_PATH = fix_path("./config.json")
+TEST_FILES_PATH = fix_path("./test_files/")
+TEST_FILES_OUTPUT = fix_path("./temp/")
+
+
+def read_config():
+    # read json file with information on dataset etc.
+    with open(CONFIG_PATH) as fid:
+        config = json.load(fid)
+
+        return config
```

Comment:
Not sure if it is intended for the return to be nested within the with statement?